build/ci-tf-e2e.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+WORKDIR /root
+
+# oc
+RUN curl -Ls https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz |tar -C /usr/local/bin -xzf - oc
+
+# ocm
+RUN yum install -y wget &&\
+    wget https://github.com/openshift-online/ocm-cli/releases/download/v0.1.66/ocm-linux-amd64 -O /usr/local/bin/ocm && \
+    chmod +x /usr/local/bin/ocm
+
+# go
+RUN curl -Ls https://go.dev/dl/go1.18.linux-amd64.tar.gz |tar -C /usr/local -xzf -
+ENV PATH="/usr/local/go/bin:${PATH}"
+ENV GOPATH=/usr/local/go
+ENV TEST_OFFLINE_TOKEN=""
+
+# terraform-provider-ocm repo
+RUN yum install -y git &&\
+    git config --global http.sslVerify false &&\
+    git clone https://github.com/terraform-redhat/terraform-provider-ocm.git --branch main
```

Comment:
why you change the WORKDIR (it is set in the beginning to /root) ?
what does this COPY . . mean?

Reply:
changing WORKDIR is just a convenience (I think we can pass it for now)
``COPY . .`` means "take the current working directory and copy it into the container when you're creating it" which is what we want

Reply:
``COPY . .`` will copy from the local directory to the container
it's instead of ``git clone``, not in addition to

Reply:
the repo clone is done inside the container, and it being used later (in the make install, and later at run-time with the make e2e)

Reply:
so instead of clone the repo inside the container (at build time), you are assuming that the 'docker build' is running from the 'src' image, which has the repo content there.  right?